memory/src/main/java/org/opensearch/ml/memory/action/conversation/GetConversationsTransportAction.java

```diff
@@ -42,27 +46,47 @@ public class GetConversationsTransportAction extends HandledTransportAction<GetC
     private Client client;
     private ConversationalMemoryHandler cmHandler;
 
+    private volatile boolean featureIsEnabled;
+
     /**
      * Constructor
      * @param transportService for inter-node communications
      * @param actionFilters for filtering actions
      * @param cmHandler Handler for conversational memory operations
      * @param client OS Client for dealing with OS
+     * @param clusterService for some cluster ops
      */
     @Inject
     public GetConversationsTransportAction(
         TransportService transportService,
         ActionFilters actionFilters,
         OpenSearchConversationalMemoryHandler cmHandler,
-        Client client
+        Client client,
+        ClusterService clusterService
     ) {
         super(GetConversationsAction.NAME, transportService, actionFilters, GetConversationsRequest::new);
         this.client = client;
         this.cmHandler = cmHandler;
+        @SuppressWarnings("unchecked")
```

Comment:
Suggest use https://github.com/opensearch-project/ml-commons/blob/2.x/plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java#L198C10-L198C77, so no need to add `@SuppressWarnings("unchecked")`